.github/workflows/sync-fork.yml

```diff
@@ -0,0 +1,30 @@
+name: Sync to Upstream
```

Comment:
This keeps forks automatically in sync with the upstream repo. It will not have any impact on the original repository.

Reply:
Sorry, I don't understand what this is.

Reply:
These changes are unrelated to the PR. Please, remove it from the PR and open a separate PR or an issue. Otherwise, the PR looks good to me.